### PR TITLE
rebase stuff

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Send/index.tsx [hunk 1]
```diff
@@ -128,8 +128,8 @@ export const CustodyToAccountMessage = ({
       return (
         <CustomBlueCartridge>
           <FormattedMessage
-            id='modals.send.firststep.fromcustody1'
-            defaultMessage='At this time, Blockchain.com only allows sending from your {coin} Trading Wallet to your {coin} Wallet.'
+            id='modals.send.firststep.fromcustody11'
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/Send/index.tsx:131]: lol

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/BankDetails/template.tsx [hunk 2]
```diff
@@ -91,7 +91,8 @@ const Template: React.FC<InjectedFormProps<{}, Props> & Props> = props => {
             {bankAccountName}
           </Text>
           <Text size='24px' color='grey600' weight={500}>
-            {achDetails && achDetails.bankAccountType.toLowerCase()}{' '}
+            {(achDetails && account?.details?.bankAccountType?.toLowerCase()) ||
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/BankDetails/template.tsx:94]: I wonder if this won't work for ACH accounts?

File: packages/blockchain-wallet-v4-frontend/src/modals/Btc/AddBtcWallet/template.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ const AddBtcWallet = ({
     <Modal size='large' position={position} total={total}>
       <Form onSubmit={handleSubmit}>
         <Wrapper>
-          <ModalHeader icon='arrow-up-circle' onClose={close}>
+          <ModalHeader icon='arrow-up-circle' onClose={() => close()}>
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/modals/Btc/AddBtcWallet/template.js:41]: Why do we need the anonymous function here?

**Review by blockdylanb — APPROVED** [review]
Just a couple small things that I can look into.

File: packages/blockchain-wallet-v4-frontend/src/modals/Btc/AddBtcWallet/template.js (hunk 3 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Btc/AddBtcWallet/template.js:41]: it didn't worked other way, I didn't spent decent time to figure it out why

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/BankDetails/template.tsx (hunk 2 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/BankDetails/template.tsx:94]: we can improve, idk tbh

**Pull request merged**